src/commands/scanner/run.ts

```diff
@@ -4,6 +4,7 @@ import {AnyJson} from '@salesforce/ts-types';
 import {Controller} from '../../ioc.config';
 import {OUTPUT_FORMAT} from '../../lib/RuleManager';
 import {ScannerCommand} from './scannerCommand';
+import {TYPESCRIPT_ENGINE_OPTIONS} from '../../lib/eslint/TypescriptEslintStrategy';
```

Comment:
A little leaky of the abstraction, but we can revisit in 3.0.

Reply:
We discussed this at standup today. I investigated some other options such as IOC or intermediaries between the CLI and engines. But those options start to seem overly complicated and have their own downsides.

Pure IOC starts to leak the CLI requirements into the engines, because the engine needs to know enough to configure a command.

We could put an intermediate layer between the CLI and the Engines that help to dynamically add stuff to the CLI, but given the 2.0 timeline, this also seemed overly complex for the one flag. I will add this item to the quip doc for 3.0 planning.

Reply:
Awesome. Thanks.